### DISCUSSION TITLE
Is there a way to make the text justification change from center?

### DISCUSSION THREAD

**Discussion by PreetM07** [SubtitleEdit/subtitleedit#4967] [Q&A] [answered]
So as new features are popping in the industry, this one is the recent trend. 

Use case: show subs/captions below the speaker.
For this the most useful feature is alignment and justification.
I know there is a feature in SE to align the text.

But is there anything related to change the justification?
Any help would be much appreciated.

**Comment by OmrSi** [comment, marked as answer]
SRT files don't really have justification or alignment.
Setting the alignment using `{\anN}` tag is actually using ASSA tags to set the position of the subtitle on the screen.
When setting it to a position on the right, meaning when N is in [3, 6, 9], the text is right-justified as well.
When setting it to a position on the left, meaning when N is in [1, 4, 7], the text is left-justified as well.

So, you can't set the justification alone for the text, as far as I know.

Although, if you only want to see it justified inside SE just for you, you can change `MpvExtraOptions` in the settings file to `<MpvExtraOptions>--sub-ass-justify=yes --sub-justify=left</MpvExtraOptions>`, you have to be using MPV as your video player and, again, this will only apply inside SE.
